### PR TITLE
Return MAC Address from network interface

### PR DESCRIPTION
**Pull request by palmtreefrb**
## Checklist:

- [x] I've placed my contribution in the appropriate section.
- [x] I've choosen a good name for it and I've checked the repo for duplicates.
- [x] I've added a readme.md following the provided template (TBD). 
- [x] My code follows the code style of this project.
- [x] I've tidy my files inside the contribution folder.
- [x] I've edited the repo README.md to include the mention and link to my contribution. It's under the proper section and the link is correct.
- [x] The code builds.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nanoframework/nf-Community-Contributions?pullRequest=30) <br/>All CLA requirements met.

**Review by josesimoes — CHANGES_REQUESTED** [review]
Looks good, just a couple of changes please:

1) Create a new folder inside Utilities, called... "Read MAC address" or similar.
2) Move the source code of the Helpers.cs class into that one.
3) Add a README.md with a title and a short explanation on the goal and a couple of code line showing how to use it.

Thanks!

File: README.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ These contributions are **NOT** supported by the **nanoFramework** team and any
 ### Utilities
 <table>
  <tr>
-  <td><!--<a href="Utility/util1">Utility One</a>--></td>
-  <td><!--<a href="Utility/util2">Utility Two</a>--></td>
+  <td><!--<a href="Utility/Helpers.cs">Utility to Return MAC Address from network interface.</a>--></td>
```
Comment (josesimoes) [inline README.md]: Please add the new folder where this is now.

File: utilities/Read-MAC-Address/Helpers.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        /// <summary>
+        /// Return String from byte[].
+        /// </summary>
+        /// <param name="barray">The value to convert. Supported types are: byte[].</param>
+        /// <returns>String from byte array</returns>
+        /// <remarks>Usage: string b = Utilities.ByteArrayToHex(new byte[] { 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x20 });.</remarks>
+
+        public static string ByteArrayToHex(byte[] barray)
+        {
+            char[] c = new char[barray.Length * 2];
```
Comment (josesimoes) [inline utilities/Read-MAC-Address/Helpers.cs]: There is a much easier way of doing this.
Check this code here: https://github.com/nanoframework/Samples/blob/62f77443c624143883a333f4c8a24c193d0f00e0/samples/ToStringTest/ToStringTestingShared/TestCode.cs#L54

Comment (palmtreefrb) [reply utilities/Read-MAC-Address/Helpers.cs]: Yes easier, but is it unit testable ;-) 
Which method do you prefer I change it in, ByteArrayToHex or GetMacId ?

Comment (josesimoes) [reply utilities/Read-MAC-Address/Helpers.cs]: Since you already have that method in place just use the ByteArrayToHex .

**Review by josesimoes — APPROVED** [review]
Perfect! Thanks for sharing. 👍 😃

**Comment by josesimoes** [comment]
@palmtreefrb please sign the CLA so this can be merged

**Review by josesimoes — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**